www/source/partials/docs/_using-hab-packages.html.md.erb

```diff
@@ -29,14 +29,14 @@ If your host machine is running Linux, do the following to run your packages:
 * Run the `hab` CLI as root.
 
     ```shell
-    $ sudo hab start yourorigin/yourname
+    $ sudo hab svc start yourorigin/yourname
     ```
 
-You may use the same `hab start` command on Windows but omit the `sudo` command. However, you should be inside of an elevated shell. Also, note that the `hab` user is not necessary on Windows. If it is absent, services will run under the identity of the current user. If a `hab` user is present, you will need to provide its password via the`--password` argument:
+You may use the same `hab svc start` command on Windows by omitting the `sudo` command. However, you should be inside of an elevated shell. Also, note that the `hab` user is not necessary on Windows. If it is absent, services will run under the identity of the current user. If a `hab` user is present, you will need to provide its password via the`--password` argument:
```

Comment:
This has already been correctly updated to `hab run`. The "but omit" → "by omitting" change doesn't make sense to me, so I think we should leave it as is.